project-base/app/assets/js/admin/grapesjs/plugins/grapesjs-products-plugin.js

```diff
@@ -74,9 +74,17 @@ export default grapesjs.plugins.add('products', editor => {
         const productName = attributes[dataProductName];
 
         if (product && productName) {
+            const templateName = productName === unkonwnProductName ? 'grapejs-unknown-product-card' : 'grapejs-product-card';
+
             element.appendChild(createDiv(`
-                <div style="margin-top: 2em; text-align: center;">${productName}<br/><br/>${product}</div>
-                <img src="${window.location.origin}/public/admin/images/grapejs-product-list-item.png" alt="${dataProductName}" style="max-width: 100%;/>
+                <div style="position: relative; max-width: 100%;">
+                    <div class="gjs-products-product-card-name">
+                        ${productName.substring(0, 55)}
+                        <br><br>
+                        ${product}
```

Comment:
nitpick - I would rename the variable to `productCatnum` to make it more clear

Reply:
renamed, good idea